Set AccordionHeader's background color to align with Figma (#4555)
jpmorganchase/salt-ds: .changeset/smart-jobs-heal.md

---
"@salt-ds/core": patch
---

> Sets the `AccordionHeader`'s background color as `transparent` to align with Figma. Closes #4544.

Reviewer: ```suggestion
Updated `AccordionHeader`'s background color to `transparent` to align with Figma. Closes #4544.
```

Author: Done ✅
